server/src/dtos/person.dto.ts

```diff
@@ -218,11 +218,13 @@ export class PeopleResponseDto {
   hasNextPage?: boolean;
 }
 
+const asDateString = (x: string | Date | null) => (x instanceof Date ? x.toISOString().split('T')[0] : x);
```

Comment:
Is this a dupe of the utils/date.ts below?

Reply:
My bad, fixed